django_rest/django_rest/serialyzers.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.hashers import make_password
+from django_rest.models import Employee
+from rest_framework import serializers
+from django.contrib.auth.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['username', 'first_name', 'last_name', 'email']
+
+
+class EmployeeSerializer(serializers.Serializer):
+    username = serializers.CharField(required=False, allow_blank=True, max_length=100)
```

Comment:
Do we need `allow_blank = True`?
And username should be `required = True` I guess.